config/boot.rb

```diff
@@ -35,15 +35,15 @@
 # Load Resque configuration and controller
 require 'resque'
 Resque.redis = (defined? REDIS_URL) ? REDIS_URL : "localhost:6379/resque:#{ENV['ROBOT_ENVIRONMENT']}"
-# begin
-#   if defined? REDIS_TIMEOUT
-#     _server, _namespace = REDIS_URL.split('/', 2)
-#     _host, _port, _db = _server.split(':')
-#     _redis = Redis.new(host: _host, port: _port, thread_safe: true, db: _db, timeout: REDIS_TIMEOUT.to_f)
-#     Resque.redis = Redis::Namespace.new(_namespace, redis: _redis)
-#   else
-#     Resque.redis = REDIS_URL
-#   end
-# end
+begin
+  if defined? REDIS_TIMEOUT
+    _server, _namespace = REDIS_URL.split('/', 2)
+    _host, _port, _db = _server.split(':')
+    _redis = Redis.new(host: _host, port: _port, thread_safe: true, db: _db, timeout: REDIS_TIMEOUT.to_f)
+    Resque.redis = Redis::Namespace.new(_namespace, redis: _redis)
+  else
+    Resque.redis = REDIS_URL
```

Comment:
Agree this `else` was always useless.  Not sure about the namespace and `REDIS_TIMEOUT` though.  Are we sure those are disposable?

Reply:
Hmm - I just copied boilerplate;  I just renamed all vars to NOT start with underscore as they are all used.